src/components/Navigation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import userPic from '../multimedia/user.png';
+
+function Navigation() {
+  return (
+    <nav id="navbar">
+      <div id="nav-links">
+        <h1>Bookstore CMS</h1>
+        <NavLink to="/" className="link">Books</NavLink>
+        <NavLink to="/categories" className="link">Categories</NavLink>
```

Comment:
- please create a `categories` component with the text `check status` for now.

Reply:
You are blocking me from the approval since you're not reading the project requirements. What you're writing here is not necessary at all. this is what is required.

Initialise React app.
The building blocks of your app should be set as re-usable components. Create a directory for your components.
Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
**Books - the default view:**
Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
Should display a form for adding a book (no functionality yet).
Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
**Categories:**
Should display a button "Check status" only.
Styling is not required at this point.

What is **Bolded** are pages that have to display what is after the ":"

In the categories, which is not a component but a page or view, what is asked is **"Should display a button "Check status" only."**